test/integration/export/test/ssr.js

```diff
@@ -48,14 +48,12 @@ export default function (context) {
       expect(html).toMatch(/Query is: {}/)
     })
 
-    it('should render _error on 404', async () => {
-      const html = await renderViaHTTP(context.port, '/404')
-      expect(html).toMatch(/404/)
-    })
-
-    it('should export 404.html instead of 404/index.html', async () => {
```

Comment:
I wasn't sure if creating a new test-suite for this test alone would be worth it considering the added time for CI etc. I think a `export-default` suite with more tests than this one would make sense though.